hazelcast/src/test/java/com/hazelcast/map/PartitionPredicateTest.java

```diff
@@ -206,10 +205,20 @@ public void testSerialization() {
         Data serialized = serializationService.toData(predicate);
         PartitionPredicate deserialized = serializationService.toObject(serialized);
 
-        assertEquals(partitionKey, deserialized.getPartitionKey());
+        assertEquals(partitionKey, deserialized.getPartitionKeys().stream().findFirst().get());
         assertEquals(Predicates.alwaysTrue(), deserialized.getTarget());
     }
 
+    @Test
+    public void partitionPredicateWithNullKeyThrowsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> Predicates.partitionPredicate(null, a -> true));
+    }
+
+    @Test
+    public void partitionPredicateWithNullPredicateThrowsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> Predicates.partitionPredicate("foo", null));
+    }
+
```

Comment:
should we add the same tests for multi partition predicate as well?

Reply:
I already have test for these in `PartitionsPredicateTest.java`

I'm renaming the class to `MultiPartitionPredicateTest.java` so look for `multiPartitionPredicateWithNullSetThrowsNullPointerException` on my next push